urgent-care-api/src/main/java/gov/va/api/health/urgentcare/api/CoverageApi.java

```diff
@@ -0,0 +1,92 @@
+package gov.va.api.health.urgentcare.api;
+
+import gov.va.api.health.urgentcare.api.resources.Coverage;
+import gov.va.api.health.urgentcare.api.resources.OperationOutcome;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+public interface CoverageApi {
+  /* @Operation(
+      summary = "Coverage Read",
+      description =
+          "https://www.hl7.org/fhir/R4/coverage.html",
+      tags = {"Coverage"}
+  )
+  @GET
+  @Path("Coverage/{id}")
+  @ApiResponse(
+      responseCode = "200",
+      description = "Record found",
+      content =
+      @Content(
+          mediaType = "application/fhir+json",
+          schema = @Schema(implementation = Coverage.class)
+      )
+  )
+  @ApiResponse(
+      responseCode = "400",
+      description = "Not found",
+      content =
+      @Content(
+          mediaType = "application/json+fhir",
+          schema = @Schema(implementation = OperationOutcome.class)
+      )
+  )
+  @ApiResponse(
+      responseCode = "404",
+      description = "Bad request",
+      content =
+      @Content(
+          mediaType = "application/json+fhir",
+          schema = @Schema(implementation = OperationOutcome.class)
+      )
+  )
+  Coverage coverageRead(
+      @Parameter(in = ParameterIn.PATH, name = "id", required = true) String id); */
+
+  @Operation(
+    summary = "Coverage Search",
+    description = "https://www.hl7.org/fhir/R4/coverage.html",
+    tags = {"Coverage"}
+  )
+  @GET
+  @Path("Coverage")
+  @ApiResponse(
+    responseCode = "200",
+    description = "Record found",
+    content =
+        @Content(
+          mediaType = "application/json+fhir",
+          schema = @Schema(implementation = Coverage.Bundle.class)
+        )
+  )
+  @ApiResponse(
+    responseCode = "400",
+    description = "Not found",
```

Comment:
These descriptions are backwards. 400 is bad request

Reply:
👍